### ISSUE TITLE
[Enhancement]: Change colormap on Stage 3

### ISSUE THREAD

**Issue by arikaegan** [kevin218/Eureka#571]
### Instrument

MIRI (Stages 1-3), Light curve fitting (Stages 4-6)

### What is your suggestion?

It would be helpful if the colormap on Background Subtracted Frame images (e.g. fig3301_file0_int00002_ImageAndBackground.png) was a diverging colormap centered at 0. That way, it would be easier to compare the top and bottom plots to each other, as well as use the Background plot as a diagnostic for what background was measured and removed. As it is now, the 0-points on both plots within the figure sit at different colors due to the different vmin/vmax values and it's challenging to (1) distinguish the 0-point from lower/higher values and (2) compare the plots with each other.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by arikaegan** [comment]
I took the liberty of exploring this. I found a nice answer on re scaling diverging colormaps on StackExchange and added that function to plots_s3.py in S3_data_reduction. It works quite well. In the .zip file, there's a modified script called plots_s3_ae.py with that change implemented. I chose the BrBG (brown/green) diverging color map and updated it for 3301 and 3308 plots. Example plots of those are in the .zip too.
